nvm.sh

```diff
@@ -1044,6 +1044,17 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ "_$NVM_OS" = "_linux" ]; then
+    MAKE_JOBS="$(nproc)"
+  elif [ "_$NVM_OS" = "_freebsd" ] || [ "_$NVM_OS" = "_darwin" ]; then
+    MAKE_JOBS="$(sysctl -n hw.ncpu)"
+  elif [ "_$NVM_OS" = "_sunos" ]; then
+    MAKE_JOBS="$(psrinfo | wc -l)"
+  else
+    MAKE_JOBS="1"
```

Comment:
Is there a chance that any of the other commands could fail, making `MAKE_JOBS` be the empty string, and not hitting this `else` block?